kpi-tracker/src/main.rs

```diff
@@ -85,32 +110,53 @@ struct ContractInstanceDetails {
     block_hash: BlockHash,
 }
 
+/// Represents a compound unique constraint for relations between accounts and
+/// transactions
```

Comment:
I don't understand what this means. What is a "compound unique constraint"? Does this mean that the address is affected by the transaction?

Reply:
Maybe its phrased badly. What I mean is, that every row is unique when looking at the combination of address/transaction hash. Again this is a reference to how it'll be implemented in SQL. I'll remove the "compound unique constraint" part, as it probably just confuses more than explains anything.